polymorphic/tests/test_orm.py

```diff
@@ -360,7 +360,11 @@ def test_manytomany_field(self):
         # no pretty printing
         ModelShow1_plain.objects.create(field1='abc')
         ModelShow2_plain.objects.create(field1='abc', field2='def')
-        self.assertEqual(qrepr(ModelShow1_plain.objects.all()), '<QuerySet [<ModelShow1_plain: ModelShow1_plain object>, <ModelShow2_plain: ModelShow2_plain object>]>')
+        # repr classnames are not hardcoded in Django 1.11+
+        if django.VERSION >= (1, 11):
+            self.assertEqual(qrepr(ModelShow1_plain.objects.all()), '<PolymorphicQuerySet [<ModelShow1_plain: ModelShow1_plain object>, <ModelShow2_plain: ModelShow2_plain object>]>')
+        else:
+            self.assertEqual(qrepr(ModelShow1_plain.objects.all()), '<QuerySet [<ModelShow1_plain: ModelShow1_plain object>, <ModelShow2_plain: ModelShow2_plain object>]>')
```

Comment:
I wasn't sure about repeating the whole line (I thought it wasn't very DRY), but I noticed that this approach was favoured in other similar parts of this file. If you'd prefer it changed to be more DRY, please let me know.